pyproject.toml

```diff
@@ -25,14 +25,15 @@ classifiers = [
 ]
 requires-python = '>="3.8"'
 dependencies = [
-    'pymatgen>=2022.7.8',
+    'pymatgen>=2022.7.19',
     'custodian>=2019.8.24',
     "pydantic",
     "monty",
     'jobflow>=0.1.5',
     "PyYAML",
     "numpy",
     "click",
+    "pymatgen-analysis-defects>=2022.8.24",
```

Comment:
Can you add this as an optional dependency. Basically, pymatgen-analysis-defects relies on dscribe which is rather heavy.